pkg/storage/system_intake.go

```diff
@@ -219,7 +219,7 @@ func (s *Store) FetchSystemIntakeByID(ctx context.Context, id uuid.UUID) (*model
 	const idMatchClause = `
 		WHERE system_intakes.id=$1
 `
-	err := s.db.Get(&intake, fetchSystemIntakeSQL+idMatchClause, id)
+	err := s.db.GetContext(ctx, &intake, fetchSystemIntakeSQL+idMatchClause, id)
```

Comment:
To get the benefit of the instrumentation, we need to use the `XXXContext(...)` variants, which we should arguably be doing anyways to get use context cancellation.

Effectively, this is a single-line change that would have to be done throughout the `storage` layer, but since we already have `context` in almost everything, it'd be a simple lift.